index.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 const UNITS = ['B', 'kB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'];
 
-module.exports = num => {
+module.exports = (num, localize) => {
```

Comment:
Forgot to mention this last time, but could you make an option object as the second argument, e.g. `{localize: true}`.